### ISSUE TITLE
Application specified OS.js instances

### ISSUE THREAD

**Issue by leopck** [os-js/OS.js#735]
Right now, it seems that every user has a shared OS.js from a single node.js server.

Meaning, if you installed app1 inside the node.js server, it'll reflect to all users on OS.js and if each users keeps installing applications this would reflect to other users as well.

I was thinking that having an OS that only retains for a particular user or a user specific configuration of a OS would be great for a "roaming OS".

tl;dr 
So every user that installs an application wouldn't affect other users, and each time the user login back in, all the applications that was pre-installed will be loaded back once more.

**Comment by andersevenrud** [comment]
Yes, at the moment in v3 there's only global package support that's entirely relying on the server hosting the solution.

I have started on integration of per-user applications that loads packages from the internal VFS, meaning that the user can install packages to any arbitrary location (even remote).

I've created a new issue for this here: https://github.com/os-js/osjs-client/issues/106 (as well as linked any related issues).
